src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2797,7 +3002,8 @@ because xbuild doesn't support framework reference assemblies.
 	<Delete Files="$(_AndroidStaticResourcesFlag)" />
 	<Delete Files="$(_AndroidLibraryProjectImportsCache)" />
 	<Delete Files="$(_AndroidLibrayProjectAssemblyMapFile)" />
-	<Delete Files="$(IntermediateOutputPath)R.txt" />
+	<Delete Files="$(_AndroidAapt2VersionFile)" />
+	<Delete Files="$(MonoAndroidIntermediate)R.txt" />
```

Comment:
This should also be `$(IntermediateOutputPath)`. There are no other occurrences of `$(MonoAndroidIntermediate)R.txt`, but 3 of `$(IntermediateOutputPath)R.txt`.